services/workflows-service/src/alert/alert.service.ts

```diff
@@ -13,8 +13,14 @@ import { AlertDefinitionRepository } from '@/alert-definition/alert-definition.r
 import { InlineRule } from '@/data-analytics/types';
 import _ from 'lodash';
 import { AlertExecutionStatus } from './consts';
+import { computeHash } from '@/common/utils/sign/sign';
+import { TDedupeStrategy, TExecutionDetails } from './types';
 import { findByKeyCaseInsensitive } from '@/common/schemas';
 
+const DEFAULT_DEDUPE_STRATEGIES = {
+  cooldownTimeframeInMinutes: 1440,
```

Comment:
Make it easier to understand :)
```suggestion
  cooldownTimeframeInMinutes: 60 * 24,
```